exporters/otlp/otlpmetric/otlpmetricgrpc/README.md

```diff
@@ -0,0 +1,42 @@
+# OpenTelemetry-Go OTLP Metric gRPC Exporter
+
+[![Go Reference](https://pkg.go.dev/badge/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc.svg)](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc)
+
+[OpenTelemetry Protocol Exporter](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.20.0/specification/protocol/exporter.md) implementation using gRPC.
+
+## Installation
+
+```
+go get -u go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc
+```
+
+## Usage
+
+Exporters should be created using the [New](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlpmetric/otlpmetricgrpc#New) and used with a [PeriodicReader].
+
+## Environment Variables
+
+The following environment variables can be used (instead of options objects) to override the default configuration.
```

Comment:
If both options and env vars are specified, which take precedence?

Reply:
options (it should be documented)